airbyte-webapp/src/components/JobItem/components/JobLogs.tsx

```diff
@@ -53,7 +53,7 @@ const jobIsSynchronousJobRead = (job: SynchronousJobRead | JobsWithJobs): job is
 export const JobLogs: React.FC<JobLogsProps> = ({ jobIsFailed, job }) => {
   const isSynchronousJobRead = jobIsSynchronousJobRead(job);
 
-  const id: number | string = (job as JobsWithJobs).job?.id ?? (job as SynchronousJobRead).id;
+  const id: number | string = job.job?.id ?? (job as SynchronousJobRead).id;
```

Comment:
Please remove this file from your PR.

Reply:
I think the repo has some github actions that added these changes.  I have reverted them.